ghost/email-service/test/batch-sending-service.test.js

```diff
@@ -461,7 +461,7 @@ describe('Batch Sending Service', function () {
             sinon.assert.callCount(sendBatch, 101);
             const sendBatches = sendBatch.getCalls().map(call => call.args[0].batch);
             assert.deepEqual(sendBatches, batches);
-            assert.equal(maxRunningCount, 10);
+            assert.equal(maxRunningCount, 2);
```

Comment:
Why not use the same const as before?

const MAX_SENDING_CONCURRENCY = 2; at batch-sending-service.js